src/builders/JsonapiResourceWriter.js

```diff
@@ -0,0 +1,193 @@
+import { merge, mergeWith, isEmpty, isArray, flatMap, forOwn } from 'lodash';
+import Uuid from 'uuid/v4';
+
+import JsonapiResourceBuilder from './JsonapiResourceBuilder';
+
+const concatNestedArrayCustomizer = (a, b) => {
+  if (isArray(a)) {
+    return a.concat(b);
+  };
+};
+
+/* eslint-disable no-param-reassign */
```

Comment:
Pourquoi ne pas transformer cette fonction en méthode privée ? 🤔 ça évitera une fonction qui fait des mutations sur ces propres valeurs d'entrée

Reply:
Le pb c'est que j'utilise pas de this. Je vais essayer de tricher mais perso je trouve ça très dégueu, ça me forcerait sinon à faire des `this[iVarName]`